### PR TITLE
[Tests] Add test for Multus and OVS CNI

### PR DESCRIPTION
**Pull request by myakove**
Add test for Multus and OVS CNI.

Create VMs with OVS network via Multus and check connectivity.

### CODE REVIEW

File: tests/multus_connectivity_test.go [hunk 1]
```diff
@@ -0,0 +1,171 @@
...
+			for i := range vmiList{
+				err := ktests.CheckForTextExpecter(vmiList[i], []expect.Batcher{
+					&expect.BSnd{S: fmt.Sprintf("ping -w 3 %s \n", vmsIp[1-i])},
+					&expect.BExp{R: "3 packets received"},
+				}, 60)
+				Expect(err).ToNot(HaveOccurred())
+			}
+		})
+	})
+})
```
Comment (duyanyan) [inline tests/multus_connectivity_test.go]: Looks good!!! How about adding some test for the connectivity of the vms' default network to make sure the it won't be affected by the extra network?

**Comment by gbenhaim** [comment]
@myakove Can you please rebase?

**Comment by gbenhaim** [comment]
@myakove The ovs-vsctl-pod manifest should be fixed:

```
SSSSS{"command":"/usr/bin/oc create -f tests/manifests/ovs-vsctl-pod.yml -n kubevirt-test-default","component":"","level":"error","msg":"command failed: cannot run command","output":"","pos":"utils.go:2004","reason":"exit status 1","stderr":"error: the namespace from the provided object \"kube-system\" does not match the namespace \"kubevirt-test-default\". You must pass '--namespace=kube-system' to perform this operation.\n","timestamp":"2018-11-06T14:56:07.387113Z"}
```

**Comment by myakove** [comment]
ci test please

File: tests/manifests/ovs-vsctl-pod.yml [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+        operator: Exists
+        effect: NoSchedule
+      containers:
+      - name: ovs-vsctl-pod
+        command:
+        - /bin/bash
+        - -c
+        - |
+          sleep infinity
+        image: brew-pulp-docker01.web.prod.ext.phx2.redhat.com:8888/openshift3/ose-node:v3.11.0-0.32.0.0
```
Comment (gbenhaim) [inline tests/manifests/ovs-vsctl-pod.yml]: Can you use u/s image?

File: tests/multus_connectivity_test.go [hunk 3]
```diff
@@ -0,0 +1,223 @@
...
+	var vmiList [2]*v1.VirtualMachineInstance
+	var ovsVmsIp [2]string
+	var nodesNames [2]string
+	var defaultVmsIp [2]string
+	var nodesToIpList [2]NodesToIp
+	privilegedTestUser := "privileged-test-user"
+
+	BeforeEach(func() {
+		ovsVmsIp = [2]string{"192.168.0.1", "192.168.0.2"}
+		_, _, err := ktests.RunCommand("oc", "create", "serviceaccount", privilegedTestUser)
```
Comment (gbenhaim) [inline tests/multus_connectivity_test.go]: You can reuse the test_admin that the other tests are using. It's details can be found in util.go.

Comment (myakove) [reply tests/multus_connectivity_test.go]: What do you mean? 
I need to run a privileged pod on the test namespace.
Can I use test_admin for 
serviceAccount and serviceAccountName in the pod yaml?

**Comment by myakove** [comment]
ci test please

**Comment by myakove** [comment]
ci test please

**Comment by cynepco3hahue** [comment]
ci test please

**Comment by myakove** [comment]
ci test please

**Comment by myakove** [comment]
ci test please

**Pull request merged**